website/public/index.html

```diff
@@ -207,9 +207,50 @@
       }
       // End Rollbar Snippet
     </script>
+
+    <script
+      id="unityLoader"
+      src="%PUBLIC_URL%/unity/Build/DCLUnityLoader.js?%REACT_APP_LOADER_VERSION%"
+    ></script>
+    <script>
+      var script = document.getElementById("unityLoader");
+      script.onload = () => {};
+      var UnityLoader = UnityLoader || {
+        instantiate: function (dclCanvas) {
+          var canvas = document.createElement("canvas");
+          canvas.addEventListener("contextmenu", function (e) {
+            e.preventDefault();
+          }),
+            (canvas.id = "#canvas");
+          dclCanvas.appendChild(canvas);
+
+          return new Promise((resolve) => {
+            var buildUrl = "%PUBLIC_URL%/unity/Build";
+            var loaderUrl = buildUrl + "/unity.loader.js";
+            var config = {
+              dataUrl: buildUrl + "/unity.data.unityweb",
+              frameworkUrl: buildUrl + "/unity.framework.js.unityweb",
+              codeUrl: buildUrl + "/unity.wasm.unityweb",
+              streamingAssetsUrl: "StreamingAssets",
+              companyName: "Decentraland",
+              productName: "Decentraland World Client",
+              productVersion: "0.1",
+            };
```

Comment:
Please move this into the initializeRenderer function, you must replace `%PUBLIC_URL%` by `baseUrl`